### PR TITLE
Link to the "Known Issue" note next to references of Perfetto.

### PR DESCRIPTION
**Pull request by dgaliffiAMD**

### CODE REVIEW

File: docs/how-to/understanding-omnitrace-output.rst [hunk 1]
```diff
@@ -311,16 +311,17 @@ set ``OMNITRACE_OUTPUT_PREFIX="%argt%-"``, and let Omnitrace cleanly organize th
...
 
 Perfetto output
 ========================================
 
-Use the ``OMNITRACE_OUTPUT_FILE`` to specify a specific location. If this is an 
+Use the ``OMNITRACE_OUTPUT_FILE`` to specify a specific location. If this is an
 absolute path, then all ``OMNITRACE_OUTPUT_PATH`` and similar
-settings are ignored. Visit `ui.perfetto.dev <https://ui.perfetto.dev>`_ and open this file.
+settings are ignored. Visit `ui.perfetto.dev <https://ui.perfetto.dev>`_ and open
+this file (See `NOTE <https://github.com/ROCm/omnitrace/blob/main/README.md>`_).
```
Comment (peterjunpark) [inline docs/how-to/understanding-omnitrace-output.rst:324]:
```suggestion
this file.

.. important::

   Perfetto validation is done with trace_processor v46.0 as there is a known issue with v47.0.
   If you are experiencing problems viewing your trace in the latest version of `Perfetto <http://ui.perfetto.dev>`_,
   then try using `Perfetto UI v46.0 <https://ui.perfetto.dev/v46.0-35b3d9845/#!/>`_.
```

IMO, "See NOTE" is kind of confusing IMO. Maybe we could just directly inform the user about the issue.

image

File: docs/what-is-omnitrace.rst [hunk 2]
```diff
@@ -12,13 +12,14 @@ instrumentation, call-stack sampling, and various other features for determining
 which function and line number are currently executing.
 
 A visualization of the comprehensive Omnitrace results can be observed in any modern
-web browser. Upload the Perfetto (``.proto``) output files produced by Omnitrace at 
-`ui.perfetto.dev <https://ui.perfetto.dev/>`_ to see the details.
+web browser. Upload the Perfetto (``.proto``) output files produced by Omnitrace at
+`ui.perfetto.dev <https://ui.perfetto.dev/>`_ to see the details
+(See `NOTE <https://github.com/ROCm/omnitrace/blob/main/README.md>`_).
```
Comment (peterjunpark) [inline docs/what-is-omnitrace.rst:17]:
```suggestion
`ui.perfetto.dev <https://ui.perfetto.dev/>`_ to see the details.

.. important::
   Perfetto validation is done with trace_processor v46.0 as there is a known issue with v47.0.
   If you are experiencing problems viewing your trace in the latest version of `Perfetto <http://ui.perfetto.dev>`_,
   then try using `Perfetto UI v46.0 <https://ui.perfetto.dev/v46.0-35b3d9845/#!/>`_.
```

File: docs/install/install.rst [hunk 3]
```diff
@@ -237,18 +237,18 @@ Installing Dyninst via Spack
...
-Any ``OMNITRACE_USE_<VAL>`` option which has a corresponding ``TIMEMORY_USE_<VAL>`` 
+Any ``OMNITRACE_USE_<VAL>`` option which has a corresponding ``TIMEMORY_USE_<VAL>``
 option means that the Timemory support for this feature has been integrated
-into Perfetto support for Omnitrace, for example, ``OMNITRACE_USE_PAPI=<VAL>`` also configures 
+into Perfetto support for Omnitrace, for example, ``OMNITRACE_USE_PAPI=<VAL>`` also configures
 ``TIMEMORY_USE_PAPI=<VAL>``. This means the data that Timemory is able to collect via this package
-is passed along to Perfetto and is displayed when the ``.proto`` file is visualized 
-in `the Perfetto UI <https://ui.perfetto.dev>`_.
+is passed along to Perfetto and is displayed when the ``.proto`` file is visualized
+in `the Perfetto UI <https://ui.perfetto.dev>`_ (See `NOTE <https://github.com/ROCm/omnitrace/blob/main/README.md>`_).
```
Comment (peterjunpark) [inline docs/install/install.rst:251]:
```suggestion
in `the Perfetto UI <https://ui.perfetto.dev>`_.

.. important::
   Perfetto validation is done with trace_processor v46.0 as there is a known issue with v47.0.
   If you are experiencing problems viewing your trace in the latest version of `Perfetto <http://ui.perfetto.dev>`_,
   then try using `Perfetto UI v46.0 <https://ui.perfetto.dev/v46.0-35b3d9845/#!/>`_.
```

**Comment by dgaliffiAMD** [comment]
Thanks, @peterjunpark . I've merged your changes and updated "understanding-omnitrace-output" as well.

**Review by peterjunpark — APPROVED** [review]

**Review by gsitaram — APPROVED** [review]
@dgaliffiAMD , looks good although I don't know how to check how this would be rendered in the HTML document. I trust your experience on that. Thank you for fixing this.

**Comment by peterjunpark** [comment]
> @dgaliffiAMD , looks good although I don't know how to check how this would be rendered in the HTML document. I trust your experience on that. Thank you for fixing this.

image
This link will show you a demo webpage @gsitaram

**Comment by gsitaram** [comment]
This is very good to learn, @peterjunpark. Thank you!

**Pull request merged**